crates/teloxide-core/src/types/chat_member.rs

```diff
@@ -623,19 +637,26 @@ impl ChatMemberKind {
     ///
     /// I.e. returns **`false`** if the user
     /// - has left or has been banned in the chat
-    /// - is restricted and doesn't have the [`can_send_media_messages`] right
+    /// - is restricted and doesn't have all send media right
     /// Returns `true` otherwise.
-    ///
-    /// [`can_send_media_messages`]: Restricted::can_send_media_messages
-    #[deprecated(
-        since = "0.9.0",
-        note = "Match manually and use `can_send_media_messages` field directly. Details: https://github.com/teloxide/teloxide/issues/781"
-    )]
```

Comment:
I don't think chat member API needs any change at least so far. I think would even be okay to remove the methods marked as deprecated here already, since it's been more than a year since they were deprecated. But, of course, granted a new minor release of `teloxide`.

Reply:
We already have so many breaking changes in master, so that is not a problem 😅